cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,53 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+// Tailscale 1.14 and before stored state in
+// C:\WINDOWS\system32\config\systemprofile\AppData\Local
```

Comment:
in %LocalAppData% (usually "C:\WINDOWS\system32\config\systemprofile\AppData\Local"
when tailscaled.exe is running as a non-user system service)

Reply:
Done.